test/common.jl

```diff
@@ -169,14 +173,9 @@ function create_system_with_dynamic_inverter()
         filt,
     ) #Output Filter
 
-    sys = System(100)
-    for bus in nodes_OMIB
-        add_component!(sys, bus)
-    end
-    for lines in branch_OMIB
-        add_component!(sys, lines)
-    end
-    add_component!(sys, battery)
+    @test get_V_ref(test_inverter) == 1.045
+    @test get_P_ref(test_inverter) == 0.01375
+    @test get_Q_ref(test_inverter) == 0.0
```

Comment:
@jd-lara I think the old code had incorrect values that were passed to DynamicInverter.

Reply:
It might be the case. I think that we never checked that the data in the static devices matched the refs in the dynamic ones.